pydoctor/templates/apidocs.css

```diff
@@ -21,6 +21,14 @@
     margin-bottom: 1.8em;
 }
 
+a[name] {
+    position: relative;
+    bottom: 70px;
+}
+a[name]:before {
+    content: "";
```

Comment:
Is this required? Just asking.

Reply:
It was probably required back when I found this workaround, but indeed when I test it now, it doesn't seem to make a difference. I'll remove it.